### PR TITLE
Matplotlib support

### PR DESCRIPTION
**Pull request by loiswells97**

### CODE REVIEW

File: src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideWorker.js [hunk 1]
```diff
@@ -219,6 +219,41 @@ const vendoredPackages = {
       });
     },
   },
+  matplotlib: {
+    before: async () => {
+      pyodide.registerJsModule("basthon", fakeBasthonPackage);
+      pyodide.runPython(`
+      import js
+
+      class Dud:
```
Comment (conorriches) [inline src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideWorker.js:228]: mind dropping in a littler comment here saying why we have this Dud class?

**Review by conorriches — COMMENTED** [review]
Looks good to me :) I see there's a failing test though which may need looking at?

**Review by danhalson — APPROVED** [review]
Looks great

**Review by danhalson — APPROVED** [review]
LGTM

**Pull request merged**